fe1-web/src/App.tsx

```diff
@@ -29,6 +30,42 @@ configureNetwork(messageRegistry, keyPairRegistry);
 // start persisting the redux state after all reducers have been registered
 persist.persist();
 
+const BuildInfo = () => {
```

Comment:
Since this is a pretty much self-contained, it might be worth putting the component in its own file?